okio/src/commonMain/kotlin/okio/internal/Sha256MessageDigest.kt

```diff
@@ -16,6 +16,7 @@
 
 package okio.internal
 
+@ExperimentalUnsignedTypes
```

Comment:
We can't ship with these, so they'll have to migrate to signed values or some other way of representing the constants eventually.

Reply:
Agreed. I’m going to keep ’em as-is for now for readability, and I’ll strip em out very soon!
Tracking so I don’t miss it. https://github.com/square/okio/issues/763